app/src/main/java/com/spoony/spoony/presentation/explore/navigation/ExploreNavigation.kt

```diff
@@ -0,0 +1,27 @@
+package com.spoony.spoony.presentation.explore.navigation
+
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.navigation.NavController
+import androidx.navigation.NavGraphBuilder
+import androidx.navigation.NavOptions
+import androidx.navigation.compose.composable
+import com.spoony.spoony.core.navigation.MainTabRoute
+import com.spoony.spoony.presentation.explore.ExploreScreen
+import kotlinx.serialization.Serializable
+
+fun NavController.navigateExplore(
+    navOptions: NavOptions? = null
+) {
+    navigate(Explore, navOptions)
```

Comment:
```suggestion
    navigate(
        route = Explore, 
        navOptions = navOptions
    )
```
P5: 별건 아니지만,, 저는 이렇게 명시해주는걸 쪼끔 더 좋아합니다ㅎㅎ